engine/admin/ambassador_pattern_linking.md

```diff
@@ -151,4 +150,4 @@ case `192.168.1.52:6379`.
     	apk add socat && \
     	rm -r /var/cache/
 
-    CMD	env | grep _TCP= | (sed 's/.*_PORT_\([0-9]*\)_TCP=tcp:\/\/\(.*\):\(.*\)/socat -t 100000000 TCP4-LISTEN:\1,fork,reuseaddr TCP4:\2:\3 \&/' && echo wait) | sh
+    CMD	env | grep _TCP= | (sed 's/.*_PORT_\([0-9]*\)_TCP=tcp:\/\/\(.*\):\(.*\)/socat -t 100000000 TCP4-LISTEN:\1,fork,reuseaddr TCP4:\2:\3 \&/' && echo wait) | sh
```

Comment:
It's general it's good practice to have the trailing newline, as it also reduces the diff if new content is added at the end